lib/puma/runner.rb

```diff
@@ -46,7 +46,7 @@ def start_control
       app = Puma::App::Status.new @launcher
 
       if token = @options[:control_auth_token]
-        app.auth_token = token unless token.empty? or token == :none
+        app.auth_token = token unless token.empty?
```

Comment:
why not "unless token = 'none'" and set the token to 'none'? a little more meaningful I think.